Content.Server/Kitchen/Components/MicrowaveComponent.cs

```diff
@@ -73,6 +74,9 @@ public sealed partial class MicrowaveComponent : Component
 
         [DataField, ViewVariables(VVAccess.ReadWrite)]
         public int Capacity = 10;
+
+        [DataField("maxItemSize")]
```

Comment:
just [DataField]

Reply:
> just [DataField]

Shouldn't we set it to some default value like "Normal"?

Reply:
no maxItemSize is setting the fields name which is the default since MaxItemSize with first letter lowercased is maxItemSize

Reply:
Got you.